src/dotnet-uninstall/MacOs/FileSystemExplorer.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo.Versioning;
+
+namespace Microsoft.DotNet.Tools.Uninstall.MacOs
+{
+    internal static class FileSystemExplorer
+    {
+        private static readonly string DotNetInstallPath = Path.Combine("/", "usr", "local", "share", "dotnet");
+        private static readonly string DotNetSdkInstallPath = Path.Combine(DotNetInstallPath, "sdk");
+        private static readonly string DotNetRuntimeInstallPath = Path.Combine(DotNetInstallPath, "shared", "Microsoft.NETCore.App");
```

Comment:
Could you tried to install 2.0, 2.1X, 2.2X and 3.0 and see what is the actual folder name. And I think we need to aggregate them in to one runtime.

Reply:
I think Lee was talking about for mac, combine aspnet with runtime to make it just "runtime". I don't it is there

Reply:
Yes, we combine ASP.NET Runtimes and Host FXRs with .NET Runtimes for macOS. This is the intended behavior. However, we still need to know which ASP.NET Runtimes and Host FXRs are installed in order to aggregate them. That's why we need the install paths for them here. If you run the tool with `dotnet uninstall list`, you will see that there are only two sections, namely SDKs and Runtimes.